libs/datamodel/connectors/sql-datamodel-connector/src/mysql_datamodel_connector.rs

```diff
@@ -164,7 +166,13 @@ impl Connector for MySqlDatamodelConnector {
         &self.constructors
     }
 
-    fn parse_native_type(&self, name: &str, args: Vec<u32>) -> Result<NativeTypeInstance, ConnectorError> {
+    fn parse_native_type(&self, name: &str, args: Vec<String>) -> Result<NativeTypeInstance, ConnectorError> {
+        let res = parse_u32_arguments(args);
+        if res.is_err() {
+            return Err(res.err().unwrap());
+        }
+        let parsed_arg = res.unwrap();
```

Comment:
Could this work better like this?

```rust
let parsed_arg = parse_u32_arguements(args)?;
```

If not, why?

Reply:
yes, that's a lot better!